source/usage-examples/count.txt

```diff
@@ -11,34 +11,34 @@ The Node.js driver provides two methods for counting
 documents in a collection:
 
 - :node-api:`collection.countDocuments()
-  </Collection.html#countDocuments>` returns the number of documents in
+  <Collection.html#countDocuments>` returns the number of documents in
   the collection that match the specified query. If you specify a empty
   query document, ``countDocuments()`` returns the total number of
   documents in the collection.
 
 - :node-api:`collection.estimatedDocumentCount
-  </Collection.html#estimatedDocumentCount>` returns an
+  <Collection.html#estimatedDocumentCount>` returns an
   **estimation** of the number of documents in the collection based on
   collection metadata.
 
 ``estimatedDocumentCount()`` is faster than ``countDocuments()`` because
-the estimation is based on the collection's metadata rather than scanning
-the collection. In contrast,
+the estimation is based on the collection's metadata rather than
+scanning the collection. In contrast,
 ``countDocuments()`` takes longer to return, but provides an
 **accurate** count of the number of documents and supports specifying a
 filter. Choose the appropriate method for your workload.
 
 To specify which documents you wish to count, ``countDocuments()``
-accepts a :manual:`query </tutorial/query-documents>` parameter.
+accepts a :doc:`query </fundamentals/crud/query-document>` parameter.
 ``countDocuments()`` will only count the documents that match the
 specified query.
 
 ``countDocuments()`` and ``estimatedDocumentCount()`` support optional
 settings that affect the method's execution. Refer to the reference
 documentation for each method for more information.
 
-Depending on whether you specify a Callback method, ``countDocuments()`` and ``estimatedDocumentCount()`` behave in two
-different ways:
+Depending on whether you specify a Callback method, ``countDocuments()``
```

Comment:
While this may not be your document, I noticed that Callback is capitalized here and not in other instances.

Reply:
WIll update